docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+[id='import-files-to-panel']
+= Import XML / JSON Schema, Instance or Java Class Files
+
+Data mapping fundamentally involes morphing one data shape into another.  A user
+defines these shapes using XML schemas, JSON notation or by establishing a simple Java
+class.  Lets examine each file type independently.
```

Comment:
Replace "Lets" with "Let's".  However, I am wondering if this feature will be part of Syndesis at some point. If it will be, then this content needs to follow Red Hat conventions, which do not allow contractions. Red Hat conventions also does not want the language to be folksy. So if I were creating this content, I would say "The following sections examine each file type independently." But that kind of change would be pervasive in this file. I don't know if they are needed.